docs/rules/key-spacing.md

```diff
@@ -12,16 +12,17 @@ This rule enforces consistent spacing between keys and values in object literal
 
 This rule has an object option:
 
-* `"beforeColon": false` (default) disallows spaces between the key and the colon in object literals
-* `"beforeColon": true` requires at least one space between the key and the colon in object literals
-* `"afterColon": true` (default) requires at least one space between the colon and the value in object literals
-* `"afterColon": false` disallows spaces between the colon and the value in object literals
-* `"mode": strict` (default) enforces exactly one space before or after colons in object literals
-* `"mode": minimum` enforces one or more spaces before or after colons in object literals
-* `"align": "value"` enforces horizontal alignment of values in object literals
+* `"beforeColon": false` (default) disallows spaces between the key and the colon in object literals.
+* `"beforeColon": true` requires at least one space between the key and the colon in object literals.
+* `"afterColon": true` (default) requires at least one space between the colon and the value in object literals.
+* `"afterColon": false` disallows spaces between the colon and the value in object literals.
+* `"mode": strict` (default) enforces exactly one space before or after colons in object literals.
+* `"mode": minimum` enforces one or more spaces before or after colons in object literals.
+* `"align": "value"` enforces horizontal alignment of values in object literals.
 * `"align": "colon"` enforces horizontal alignment of both colons and values in object literals.
-* `"singleLine"` specifies a spacing style for single-line object literals
-* `"multiLine"` specifies a spacing style for multi-line object literals
+* `"align"` allows for fine-grained spacing when values are being aligned in object literals.
```

Comment:
What do you think about

> `"align"` __with an object value__ allows for fine-grained spacing...

here? Trying to show how it's different from `"value"` and `"colon"` without getting into too much detail.